azure-pipelines/templates/build.yml

```diff
@@ -0,0 +1,30 @@
+jobs:
+    - job:
+      templateContext:
+          outputs:
+              - output: pipelineArtifact
+                path: $(Build.ArtifactStagingDirectory)
+                artifact: drop
+                sbomBuildDropPath: "$(System.DefaultWorkingDirectory)"
+                sbomPackageName: "Azure Functions Node.js Programming Model"
```

Comment:
nit: Update this for durable. The README in this repo says "Durable Functions for Node.js" which sounds fine to me